lib/common/bitstream.h

```diff
@@ -39,6 +39,15 @@
 extern "C" {
 #endif
 
+#if defined (__cplusplus) || defined (__STDC_VERSION__) && __STDC_VERSION__ >= 199901L   /* C99 */
+# if __GNUC__ >= 7
+#  define FALLTHROUGH __attribute__((fallthrough))
+# else
+#  define FALLTHROUGH
+# endif
+#endif /* __STDC_VERSION__ */
```

Comment:
The build is still failing for Appveyor. The C99 check isn't necessary since we don't use the `inline` keyword. This should do the trick:

```c
#if __GNUC__ >= 7
#  define FALLTHROUGH __attribute__((fallthrough))
#else
#  define FALLTHROUGH
#endif
```